### PR TITLE
[NO-JIRA] Add "no-docker" pull test support for lib-db

### PR DESCRIPTION
**Pull request by henrique-aron**

### CODE REVIEW

**Review by Sonyque — APPROVED** [review]
I like this idea as well because I use the native PostgreSQL.

I'd love to see the kind of flag or option not to use separate commands for Docker/Non-Docker setup.

**Comment by henrique-aron** [comment]
> I like this idea as well because I use the native PostgreSQL.
> 
> I'd love to see the kind of flag or option not to use separate commands for Docker/Non-Docker setup.

Amazing idea, I'll do it!

File: scripts/heroku/db-pull-test.sh [hunk 1]
```diff
@@ -4,15 +4,29 @@ set -e
...
 GREEN='\033[0;32m'
 RED='\033[0;31m'
 
+echo "${no_docker}"
+
 if [ -z "$app" ]
   then
     echo -e "${RED}No app_name provided${NO_COLOR}"
-    echo "Usage: lib-db heroku-pull-test app_name heroku_app_name"
+    echo "Usage: lib-db heroku-pull-test app_nam5e heroku_app_name"
```
Comment (Sonyque) [inline scripts/heroku/db-pull-test.sh:29]: `app_nam5e` looks like a typo for me

**Comment by henrique-aron** [comment]
@Sonyque  Just as a final FYI. I've deeply tried to implement your suggestion in a 100% safe manner (in terms of the order of arguments and stuff (using long options for example)) but in the way it has the argument's setup, I couldn't make sure it wouldn't include breaking changes (which may not be worth it). That considered, I created a new script which is 100% backwards compatible and will still enable everybody to use the functionality only having to change couple of characters.
 
In Tours we are only going to create a new `yarn db:pull:test:no-docker` script, on top of the already existing Docker one.

**Pull request merged**